api/audit-trail-gw/package.json

```diff
@@ -7,14 +7,14 @@
     "build-tsc": "tsc --project ./",
     "build-lint": "eslint src --ext .js,.jsx,.ts,.tsx",
     "create-dist-folder": "mkdir -p ./dist",
-    "build": "npm-run-all build-tsc build-lint",
+    "build": "npm-run-all build-lint build-tsc",
     "test": "jest --detectOpenHandles",
     "test-watch": "npm run test -- --watchAll",
     "generate-secret": "ts-node src/tools/generate-secret-key",
     "start-node": "node dist/index.js server",
     "start-setup-api": "node dist/index.js setup-api",
     "start": "npm-run-all build start-node",
-    "setup-api": "npm-run-all build start-setup-api",
+    "setup-api": "npm i && npm-run-all build start-setup-api",
```

Comment:
this should be two separate commands

Reply:
This way a user has to do one step less when setting up the repo. Is there a case where a user wants to run `setup-api` without installing packeges?

Reply:
it is common to not link npm install with other scripts, please remove. You could argue the same with npm run start otherwise.